utils/mil_msgs/msg/LabeledObject.msg

```diff
@@ -0,0 +1,11 @@
+# Represents a single object labeled from LabelMe
+
+string name                   # Name/Type of object entered into labeling program
+
+string attributes             # text entered into attribute box of labeling program
+
+geometry_msgs/Point[] polygon # vertices of polygon drawn in labeling program
+                              # X,Y represent pixel cordinates in labeled image
+                              # Z is ignored, should be 0
```

Comment:
What about a bounding box in here too? It'd probably be best to compute all bounding boxes during injestion so we don't have to worry about it later.